.github/workflows/main.yml

```diff
@@ -163,26 +120,6 @@ jobs:
     if: false # ${{ !contains(github.event.pull_request.labels.*.name, 'skip-integration-tests') }}
     needs:
     - init
-    - build-collector-slim
-    - build-collector-full
+    - build-collector
     - build-test-containers
     secrets: inherit
-
-  build-support-packages:
-    uses: ./.github/workflows/support-packages.yml
```

Comment:
Can we also remove this file? Or do we want to change it to be run by the master branch as part of the `main-drivers.yml` workflow?

Reply:
Definitely the latter. I'll fix this.